Sources/Models/Color.swift

```diff
@@ -8,7 +8,7 @@
 import SWXMLHash
 
 public enum Color: IBDecodable {
-
+    
```

Comment:
Please do not add space at many place. Use tools like swiftlint to autocorrect for instance